### PR TITLE
Fixed embedded server compilation errors

### PR DESCRIPTION
**Pull request by temeo**
The cleanup in commit f6bbdfdc33458fb31ecd54a371d6999c709c2209
was not complete, obsolete wsrep API enums were still left in
wsrep_dummy.cc. This caused build failure when compiling
embedded server.

Removed obsolete and commented-out code from wsrep_dummy.cc.

### CODE REVIEW

**Comment by temeo** [comment]
@vaintroub https://github.com/MariaDB/server/pull/1025/files#diff-754f2355ea230c7fe36e70dc24d6cc03R39

**Comment by sciascid** [comment]
@SachinSetiya this pull request has the highest priority among our pending pull requests against bb-10.4-galera4. If possible merge it before the other ones, thanks!

File: sql/wsrep_dummy.cc [hunk 1]
```diff
@@ -36,7 +36,7 @@ const unsigned char* wsrep_xid_uuid(const XID*)
 }
 
 bool wsrep_prepare_key_for_innodb(THD* thd, const uchar*, size_t, const uchar*, size_t, struct wsrep_buf*, size_t*)
-{ return -1; }
+{ return 0; }
```
Comment (SachinSetiya) [inline sql/wsrep_dummy.cc]: Isnt -1 and 0 very different ? -1 will be converted to true , while 0 to false, Although it is never used but , still I will prefer 1 over 0

Comment (SachinSetiya) [reply sql/wsrep_dummy.cc]: Thanks!

**Pull request merged**